Source/CovPassApp/Source/Resources/Locale/en.lproj/Localizable.strings

```diff
@@ -330,92 +330,92 @@
 
 
 /********** certificate delete dialog **********/
 
-"dialog_delete_certificate_title" = "Möchten Sie das COVID-Zertifikat wirklich löschen?";
+"dialog_delete_certificate_title" = "Do you really want to delete this COVID certificate?";
 
-"dialog_delete_certificate_message" = "Die Daten werden rückstandslos von diesem Gerät entfernt. Die Wiederherstellung der Daten ist nicht möglich.";
+"dialog_delete_certificate_message" = "The data is removed from this device without leaving any residue. It is not possible to restore the data.";
 
-"dialog_delete_certificate_button_delete" = "Löschen";
+"dialog_delete_certificate_button_delete" = "Delete";
 
-"dialog_delete_certificate_button_cancel" = "Abbrechen";
+"dialog_delete_certificate_button_cancel" = "Cancel";
 
-"delete_result_dialog_header" = "Erfolgreich gelöscht";
+"delete_result_dialog_header" = "Deleted successfully";
 
-"delete_result_dialog_message" = "Das Digitale COVID-Zertifikat wurde erfolgreich von diesem Gerät entfernt.";
+"delete_result_dialog_message" = "The digital COVID certificate was successfully deleted from this device.";
 
 "delete_result_dialog_positive_button_text" = "OK";
 
 
 /********** error handling **********/
 
-"error_standard_unexpected_title" = "Entschuldigung";
+"error_standard_unexpected_title" = "Sorry";
 
-"error_standard_unexpected_message" = "Es ist ein unerwarteter Fehler aufgetreten. Bitte versuchen Sie es später erneut.";
+"error_standard_unexpected_message" = "An unexpected error has occurred. Please try again later.";
 
 "error_standard_unexpected_button_title" = "OK";
 
-"error_no_connection_to_server_title" = "Verbindungsfehler";
+"error_no_connection_to_server_title" = "Connection error";
 
-"error_no_connection_to_server_message" = "Bei der Verbindung zum Server ist ein Fehler aufgetreten. Bitte überprüfen Sie Ihre Internetverbindung.";
+"error_no_connection_to_server_message" = "An error occured while connecting to the server. Please check your internet connection.";
 
 "error_no_connection_to_server_button_title" = "OK";
 
-"error_scan_qrcode_cannot_be_parsed_title" = "QR-Code wurde nicht erkannt";
+"error_scan_qrcode_cannot_be_parsed_title" = "QR code was not recognised";
 
-"error_scan_qrcode_cannot_be_parsed_message" = "Der QR-Code gehört nicht zu einem Digitalen COVID-Zertifikat der EU. Sie erkennen das richtige Zertifikat in Deutschland daran, dass der Aussteller das Robert Koch-Institut ist.\n\nQR-Codes von zum Beispiel Impfbescheinigungen oder QR-Codes auf einigen Chargen-Aufklebern aus dem gelben Impfbuch können nicht hinzugefügt werden.";
+"error_scan_qrcode_cannot_be_parsed_message" = "The QR code does not belong to an EU Digital COVID Certificate. You can recognise the correct certificate in Germany by the fact that the issuer is the Robert-Koch-Institut.\n\nQR codes from, for example, vaccination certificates or QR codes on some batch numbers from the yellow vaccination book cannot be added.";
```

Comment:
"Robert-Koch-Institut" should be "Robert-Koch-Institute", see the imprint: https://www.rki.de/EN/Service/Imprint/imprint_node.html
Also, it's written without any hyphens. 

```suggestion
"error_scan_qrcode_cannot_be_parsed_message" = "The QR code does not belong to an EU Digital COVID Certificate. You can recognise the correct certificate in Germany by the fact that the issuer is the Robert Koch Institute.\n\nQR codes from, for example, vaccination certificates or QR codes on some batch numbers from the yellow vaccination book cannot be added.";
```

Reply:
I did it intentionally, since the certificates will state it as "Robert-Koch-Institut". But I don't know if it would make a big difference if I change it to the official English name.

Reply:
Okay, I would wait for feedback of the maintainers here.

Reply:
(I See you already changed this in https://github.com/Digitaler-Impfnachweis/covpass-ios/pull/28/commits/f69a8ea8f457d2289022fb8ff332e92c2b55d9fb.